### DISCUSSION TITLE
怎么在ObjC中进行运行时编辑PAG颜色

### DISCUSSION THREAD

**Discussion by PaofYueng** [Tencent/libpag#978] [Q&A] [answered]
RT，我想在运行时修改PAG中的颜

**Comment by domchen** [comment, marked as answer]
整体的颜色修改，你可以查一下iOS系统的标准API，看看有没在CALayer上可以直接加滤镜改颜色。PAG内部改颜色，只支持对实色图层修改，让设计师约定一个图层名称，运行时你通过PAGFile.getLayersByName()获取那个图层，强转为PAGSolidLayer，就有setColor属性了。
